src/ImageSharp/Common/Helpers/Numerics.cs

```diff
@@ -808,6 +809,25 @@ public static int ReduceSum(Vector256<int> accumulator)
         return Sse2.ConvertToInt32(vsum);
     }
 
+    /// <summary>
+    /// Reduces elements of the vector into one sum.
+    /// </summary>
+    /// <param name="accumulator">The accumulator to reduce.</param>
+    /// <returns>The sum of all elements.</returns>
+    [MethodImpl(InliningOptions.ShortMethod)]
+    public static int ReduceSumArm(Vector128<uint> accumulator)
```

Comment:
You can use `Vector128<T>.sum()` instead of this method. In general, try using Vector128/Vector256 API wherever possible. This would improve portability of the code and benefit from improvements to the API itself.

Reply:
The `ReduceSum` can also be refactored out.

Reply:
> The `ReduceSum` can also be refactored out.

We cannot get rid of `ReduceSum` yet, because we target `net6.0` and the `Vector128<T>.sum` was introduced with `net7.0`.
I am using `Vector128<T>.sum` for >= `Net7.0`: b0bfb0a0

Reply:
Sure, makes sense 👍